recipes/wxwidgets/all/conanfile.py

```diff
@@ -0,0 +1,522 @@
+        if self.settings.compiler == "clang":
+            tc.variables["wxBUILD_PRECOMP"] = "OFF"
+
+        # platform-specific options
+        if is_msvc(self):
+            tc.variables["wxBUILD_USE_STATIC_RUNTIME"] = "MT" in str(self.settings.compiler.runtime)
+            tc.variables["wxBUILD_MSVC_MULTIPROC"] = True
+        if self.settings.os == "Linux":
+            tc.variables["wxBUILD_TOOLKIT"] = self._gtk_version
+            tc.variables["wxUSE_CAIRO"] = self.options.cairo
+        # Disable some optional libraries that will otherwise lead to non-deterministic builds
+        if self.settings.os != "Windows":
+            tc.variables["wxUSE_LIBSDL"] = "OFF"
+            tc.variables["wxUSE_LIBICONV"] = "OFF"
+            tc.variables["wxUSE_LIBNOTIFY"] = "OFF"
+            tc.variables["wxUSE_LIBMSPACK"] = "OFF"
+            tc.variables["wxUSE_LIBGNOMEVFS"] = "OFF"
+
+        tc.variables["wxUSE_LIBPNG"] = "sys" if self.options.png != "off" else "OFF"
+        tc.variables["wxUSE_LIBJPEG"] = "sys" if self.options.jpeg != "off" else "OFF"
+        tc.variables["wxUSE_LIBTIFF"] = "sys" if self.options.tiff != "off" else "OFF"
+        tc.variables["wxUSE_ZLIB"] = "sys" if self.options.zlib != "off" else "OFF"
+        tc.variables["wxUSE_LIBLZMA"] = "sys" if self.options.liblzma != "off" else "OFF"
+        tc.variables["wxUSE_EXPAT"] = "sys" if self.options.expat != "off" else "OFF"
+        tc.variables["wxUSE_REGEX"] = "sys" if self.options.regex != "off" else "OFF"
+        tc.variables["wxUSE_NANOSVG"] = "sys" if self.options.svg != "off" else "OFF"
+
+        # wxWidgets features
+        tc.variables["wxUSE_UNICODE"] = self.options.unicode
+        tc.variables["wxUSE_SECRETSTORE"] = self.options.get_safe("secretstore")
+
+        # wxWidgets libraries
+        tc.variables["wxUSE_AUI"] = self.options.aui
+        tc.variables["wxUSE_OPENGL"] = self.options.get_safe("opengl", default=False)
+        tc.variables["wxUSE_HTML"] = self.options.html
+        tc.variables["wxUSE_MEDIACTRL"] = self.options.mediactrl
+        tc.variables["wxUSE_PROPGRID"] = self.options.propgrid
+        tc.variables["wxUSE_DEBUGREPORT"] = self.options.debugreport
+        tc.variables["wxUSE_RIBBON"] = self.options.ribbon
+        tc.variables["wxUSE_RICHTEXT"] = self.options.richtext
+        tc.variables["wxUSE_SOCKETS"] = self.options.sockets
+        tc.variables["wxUSE_STC"] = self.options.stc
+        tc.variables["wxUSE_WEBVIEW"] = self.options.webview
+        tc.variables["wxUSE_XML"] = self.options.xml
+        tc.variables["wxUSE_XRC"] = self.options.xrc
+        tc.variables["wxUSE_HELP"] = self.options.help
+        tc.variables["wxUSE_WXHTML_HELP"] = self.options.html_help
+        tc.variables["wxUSE_URL"] = self.options.protocol
+        tc.variables["wxUSE_PROTOCOL"] = self.options.protocol
+        tc.variables["wxUSE_FS_INET"] = self.options.fs_inet
```

Comment:
```suggestion
        tc.variables["wxUSE_FS_INET"] = self.options.fs_inet
        tc.cache_variables["CMAKE_POLICY_DEFAULT_CMP0077"] = "NEW"
```
For `tc.variables` to work correctly.